packages/server/src/fhircast/routes.ts

```diff
@@ -4,12 +4,12 @@ import { body, validationResult } from 'express-validator';
 import { asyncWrap } from '../async';
 import { getConfig } from '../config';
 import { invalidRequest, sendOutcome } from '../fhir/outcomes';
+import { authenticateRequest } from '../oauth/middleware';
 import { getRedis } from '../redis';
 
 export const fhircastRouter = Router();
 
-// TODO: Enable authentication for fhircast
-// fhircastRouter.use(authenticateRequest);
+fhircastRouter.use(authenticateRequest);
```

Comment:
See "https://github.com/medplum/medplum/blob/main/packages/server/src/fhir/routes.ts" for how to support both public and private routes.

```ts
const publicRoutes = Router();
fhirRouter.use(publicRoutes);

const protectedRoutes = Router().use(authenticateRequest);
fhirRouter.use(protectedRoutes);
```